code/game/objects/structures/crates_lockers/crates/gear_loadout.dm

```diff
@@ -755,6 +766,36 @@
 	new /obj/item/ammo_magazine/c45m(src)
 	new /obj/item/ammo_magazine/c45m(src)
 
+/obj/structure/closet/crate/gear_loadout/nanotrasen/fill()
+	new /obj/item/rig/nanotrasen(src)
+	new /obj/item/rig/nanotrasen(src)
+	new /obj/item/rig/nanotrasen(src)
+	new /obj/item/rig/nanotrasen(src)
+	new /obj/item/storage/box/nanotrasen_nexus(src)
+	new /obj/item/storage/box/nanotrasen_auxiliary(src)
+	new /obj/item/gun/energy/laser/shotgun(src)
+	new /obj/item/gun/energy/laser/shotgun(src)
+	new /obj/item/gun/energy/rifle(src)
+	new /obj/item/gun/energy/rifle(src)
```

Comment:
This gun is a noob trap I think. You'll need to buff it to be on par with lasrifles at minimum.

Reply:
Done in #17498